content/en/tracing/trace_collection/library_config/dotnet-core.md

```diff
@@ -152,6 +152,14 @@ For more information, see [Ingestion Mechanisms][11].<br>
 The number of traces allowed to be submitted per second (deprecates `DD_MAX_TRACES_PER_SECOND`). <br>
 **Default**: `100` when `DD_TRACE_SAMPLE_RATE` is set. Otherwise, delegates rate limiting to the Datadog Agent. <br>
 
+`DD_SPAN_SAMPLING_RULES`
+: **TracerSettings property**: `SpanSamplingRule`<br>
```

Comment:
It's not possible to set this in code. We tend to avoid changing the public API as much as possible

```suggestion
```